projects/output-and-error-handling/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+func main() {
+	c := http.Client{Timeout: time.Duration(1) * time.Second}
+
+	url := "http://localhost:8080"
+	max_tries := 5
+
+	fmt.Println("----------------------------")
+
+	for i := 0; i < max_tries; i++ {
+		resp, err := c.Get(url)
+		if err != nil {
+			fmt.Println("Error from request", err)
```

Comment:
Does this write to stdout or stderr? Why is that the right place to write to? (Same question applies to all of the prints)

Reply:
Thanks for the suggestion I have changed the `fmt.println` to use `fmt.Fprintln` and the `stderr` for errors.